src/psl/psl-node.h

```diff
@@ -42,6 +42,9 @@ typedef enum {
    P_REPEAT,
    P_PROPERTY_INST,
    P_SEQUENCE_INST,
+   P_UNION,
+   P_BUILT_IN_FUNC,
```

Comment:
```suggestion
   P_BUILTIN_FUNC,
```

I think it's more common to write "bultin" as a single word in technical usage.